src/ruby/Gemfile

```diff
@@ -0,0 +1,4 @@
+source 'https://rubygems.org'
+
+gem 'rdoc', '~>4.0'
+gem 'rspec', '~> 3.0'
```

Comment:
I see. I was under the impression that the user needs to explicitly generate the documentation after installing the gem. If it is handled by RubyGems, that's more convenient. Thanks!